stubs/docutils/docutils/transforms/__init__.pyi

```diff
@@ -1,14 +1,31 @@
-from _typeshed import Incomplete
+from collections.abc import Iterable
+from types import ModuleType
+from typing import Any
 
-from docutils.nodes import Node, document
+from docutils import ApplicationError, Component, TransformSpec, nodes
+from docutils.nodes import Node, pending
+
+class TransformError(ApplicationError): ...
 
 class Transform:
-    def __init__(self, document: document, startnode: Node | None = None): ...
-    def __getattr__(self, __name: str) -> Incomplete: ...
+    default_priority: int | None
+    document: nodes.document
+    startnode: Node | None
+    language: ModuleType
+    def __init__(self, document: nodes.document, startnode: Node | None = None) -> None: ...
+    def apply(self, **kwargs: Any) -> None: ...
 
-class Transformer:
-    def __init__(self, document: document): ...
+class Transformer(TransformSpec):
+    transforms: list[tuple[str, type[Transform], Node | None]]
+    document: nodes.document
+    applied: list[tuple[int, type[Transform], pending, Any]]
+    sorted: bool
+    components: dict[str, Component]
+    serialno: int
+    def __init__(self, document: nodes.document) -> None: ...
     def add_transform(self, transform_class: type[Transform], priority: int | None = None, **kwargs) -> None: ...
-    def __getattr__(self, __name: str) -> Incomplete: ...
-
-def __getattr__(name: str) -> Incomplete: ...
+    def add_transforms(self, transform_list: list[type[Transform]]) -> None: ...
+    def add_pending(self, pending: pending, priority: int | None = None) -> None: ...
```

Comment:
I would suggest using `nodes.NODE_TYPE` because it's clearer rather than just `NODE_TYPE`.

Reply:
have updated, though i've made an exception for `Node` because i think that's quite clear, and `nodes.Node` feels tautological